database/create_tables.sql

```diff
@@ -1,8 +1,8 @@
--- Table: tempaqua_sample.anchor_point
+-- Table: public.anchor_point
 

```

Comment:
for consitency, I think it's better to keep a distinct schema and not use the public one. What do you think @nicdub